ballista/rust/core/src/serde/logical_plan/to_proto.rs

```diff
@@ -778,43 +779,43 @@ impl TryInto<protobuf::LogicalPlanNode> for &LogicalPlan {
                     )))
                 }
             }
-            LogicalPlan::Projection {
-                expr, input, alias, ..
-            } => Ok(protobuf::LogicalPlanNode {
+            LogicalPlan::Projection(projection_plan) => Ok(protobuf::LogicalPlanNode {
```

Comment:
Same comment in the [pull request](https://github.com/apache/arrow-datafusion/pull/1326#discussion_r753618119)

Reply:
@liukun4515 My thoughts are the same as you, @xudong963 What you think？
I submit it first, and then modify it if necessary.

Reply:
The modification looks good to me.